tests/test_build_texinfo.py

```diff
@@ -112,3 +112,12 @@ def test_texinfo_escape_id(app, status, warning):
     assert translator.escape_id('Hello(world)') == 'Hello world'
     assert translator.escape_id('Hello world.') == 'Hello world'
     assert translator.escape_id('.') == '.'
+
+
+@pytest.mark.sphinx('texinfo', testroot='footnotes')
+def test_texinfo_footnote(app, status, warning):
+    app.builder.build_all()
+
+    output = (app.outdir / 'python.texi').read_text()
+    assert '@footnote{@w{' not in output
```

Comment:
It's enough to check the form of the new output. Checking the removal of the footnote label is meaningless.